src/mono/mono/utils/memfuncs.c

```diff
@@ -53,7 +53,7 @@
 #define unaligned_bytes(ptr) (_toi(ptr) & ptr_mask)
 #define align_down(ptr) ((void*)(_toi(ptr) & ~ptr_mask))
 #define align_up(ptr) ((void*) ((_toi(ptr) + ptr_mask) & ~ptr_mask))
-#if SIZEOF_VOID_P == 4
+#if SIZEOF_VOID_P == 4 || SIZEOF_VOID_P == 7
 #define bytes_to_words(n)	((size_t)(n) >> 2)
```

Comment:
Unrelated ?

Reply:
@steveisok Do you remember what this is for?

Reply:
I forgot to take it out.  Early on I was messing with the build and sometimes `SIZEOF_VOID_P` would be 7 for unknown reasons. Super weird, so we should look into how it can happen.